class DiscordBotException(Exception):
    pass


class AlreadyClosedException(DiscordBotException):
    pass
